apps/src/code-studio/components/ShareDialog.jsx

```diff
@@ -37,26 +47,137 @@ var ShareDialog = React.createClass({
     this.setState({isOpen: false});
   },
 
+  showSendToPhone: function (event) {
+    this.setState({
+      showSendToPhone: true,
+      showAdvancedOptions: false,
+    });
+    event.preventDefault();
+  },
+
+  showAdvancedOptions() {
+    this.setState({
+      showSendToPhone: false,
+      showAdvancedOptions: true,
+    });
+  },
+
+  clickExport: function () {
+    this.setState({exporting: true});
+    this.props.onClickExport().then(
+      this.setState.bind(this, {exporting: false}),
+      function () {
+        this.setState({
+          exporting: false,
+          exportError: 'Failed to export project. Please try again later.'
+        });
+      }.bind(this)
```

Comment:
I'd also just use an arrow for the second function. At the very least, I'd take the same approach for both.